nkms/crypto/api.py

```diff
@@ -27,13 +28,13 @@ def secure_random(
     :return: bytes
     """
     # TODO: Should we just use os.urandom or avoid the import w/ this?
-    return SYSTEM_RAND.getrandbits(num_bytes*8).to_bytes(num_bytes,
-                                                         byteorder='big')
+    return SYSTEM_RAND.getrandbits(num_bytes * 8).to_bytes(num_bytes,
+                                                           byteorder='big')
 
 
 def secure_random_range(
-    min: int,
-    max: int
+        min: int,
```

Comment:
Mind explaining why this is more appropriate here? PyLint doesn't warn me about anything when it's indented as it was, is this not the case?

Reply:
Hmm, strange. I actually am questioning whether what I'm doing is PEP8 then and whether PyLint will tell me about it.

There's nothing in there about the structure of:

```
def func(
    self,
    foo,
    bar
) -> bytes:
```

The reasoning in the PEP8 is because it's not distinguishable, but the definition is... more C-like?? because the colon is on a separate line entirely and the params are distinguishable. Can't find anything on that in the PEP8, though.

This is fine, just curious about that specific rule now.